python/ray/data/tests/test_tfrecords.py

```diff
@@ -444,11 +468,11 @@ def test_read_tfrecords_ignore_missing_paths(
     ]
 
     if ignore_missing_paths:
-        ds = ray.data.read_tfrecords(paths, ignore_missing_paths=ignore_missing_paths)
+        ds = read_tfrecords_with_fast_read_override(path)
```

Comment:
does the new implementation work as expected with the existing test behavior?
```suggestion
        ds = read_tfrecords_with_fast_read_override(paths, ignore_missing_paths=ignore_missing_paths)
```

Reply:
@scottjlee I don't think the fast implementation works with ignoring missing paths